src/modules/fundingManager/bondingCurve/tokens/ERC20Issuance_v1.sol

```diff
@@ -92,9 +93,8 @@ contract ERC20Issuance_v1 is IERC20Issuance_v1, ERC20, Ownable {
 
     /// @notice Sets the address of the minter.
```

Comment:
This function now serves for setting and removing. Maybe adjust the comment to reflect that change?

Reply:
Seeing that you also did that in the interface